src/Altinn.AccessManagement/appsettings.json

```diff
@@ -23,7 +23,8 @@
     "OpenIdWellKnownEndpoint": "http://localhost:5101/authentication/api/v1/openid/",
     "ProfileApiEndpoint": "http://localhost:5101/profile/api/v1/",
     "RegisterApiEndpoint": "http://localhost:5101/register/api/v1/",
-    "SubscriptionKeyHeaderName": "Ocp-Apim-Subscription-Key"
+    "SubscriptionKeyHeaderName": "Ocp-Apim-Subscription-Key",
+    "ApiAuthorizationBaseUrl": "http://localhost:5101/"
```

Comment:
There is already an ApiAuthorizationEndpoint in the appsettings.json in the main branch, not sure why it doesn't show here